libs/client-sdk/src/CMakeLists.txt

```diff
@@ -76,7 +76,7 @@ set_property(TARGET ${LIBNAME} PROPERTY
         MSVC_RUNTIME_LIBRARY "MultiThreaded$<$<CONFIG:Debug>:Debug>DLL")
 
 install(TARGETS ${LIBNAME})
-if (BUILD_SHARED_LIBS AND MSVC)
+if (LD_BUILD_SHARED_LIBS AND MSVC)
     install(FILES $<TARGET_PDB_FILE:${LIBNAME}> DESTINATION bin OPTIONAL)
```

Comment:
We don't have `cmake` tests that actually try to use the install files, which is why this didn't break CI.